portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,6 +30,12 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Dylon!</h1>");
+    ArrayList<Comment> messages = new ArrayList<Comment>();
+    messages.add(new Comment("Tom", "Great website!"));
+    messages.add(new Comment("Bill", "Excellent color palette!"));
+    messages.add(new Comment("Jack", "Love the aesthetic!"));
```

Comment:
I would recommend putting this inside a 

```
public void init() {
}
```
function so that it only gets initialized once.

Reply:
Done